docs/RtspStreamingGuide.md

```diff
@@ -0,0 +1,371 @@
+# USB Camera Device Service RTSP Streaming Guide
+
+## Contents
+
+[System Requirements](#system-requirements)  
+[How It Works](#how-it-works)  
+[Overview](#overview)  
+[Tested Devices](#tested-devices))  
+[Dependencies](#dependencies)  
+[Get the Source Code](#get-the-source-code)
+[Configuration options](#configuration-options)
+[Deploy the Service](#deploy-edgex-and-usb-device-camera-microservice)  
+[Verify the Service](#verify-service-and-device-profiles)   
+[Adding Devices using REST API](#adding-devices-using-rest-api)
+[Start Video Streaming](#start-video-streaming)
+[Shutting Down](#shutting-down)  
+[Troubleshooting](#troubleshooting)) 
+[License](#license)
+
+## System Requirements
+
+- Intel&#8482; Core&#174; processor
+- Ubuntu 20.04.4 LTS
+- USB-compliant Camera
+
+**Time to Complete**
+
+10-20 minutes
+
+**Other Requirements**
+
+You must have administrator (sudo) privileges to execute the user guide commands.
+
+## How It Works
+For an explanation of the architecture, see the [Readme](../README.md#how-it-works).
+
+## Overview
+EdgeX device service for communicating with USB cameras attached to Linux OS platforms.
+This service provides the following capabilities:
+- V4L2 API to get camera metadata.
+- Camera status
+- Video stream reference
+- FFmpeg framework to capture video frames and stream them to an RTSP server.
+- An [RTSP server](https://github.com/aler9/rtsp-simple-server) is embedded in the dockerized device service. 
+
+## Tested Devices
+The following devices have been tested with EdgeX:
+<!-- sorted alphabetically -->
+- AUKEY PC-LM1E Webcam
+- HP w200 Webcam
+- Jinpei JW-01B USB FHD Web Computer Camera
+- Logitech Brio 4K
+- Logitech C270 HD Webcam
+- Logitech StreamCam
+
+## Dependencies
+The software has dependencies, including Git, Docker, Docker Compose, and assorted command line tools. Follow the instructions below to install any dependency that is not already installed.  
+
+### Install Git
+Install Git from the official repository as documented on the [Git SCM](https://git-scm.com/download/linux) site.
+
+1. Update installation repositories:
+   ```bash
+   sudo apt update
+   ```
+
+2. Add the Git repository:
+   ```bash
+   sudo add-apt-repository ppa:git-core/ppa -y
+   ```
+
+3. Install Git:
+   ```bash
+   sudo apt install git
+   ```
+
+### Install Docker
```

Comment:
I changed with the last batch of docs, but maybe it would be best to list the steps in the document itself. My concern with this however is that they could change the steps with a future release and our docs would be outdated and it would be hard to catch.

Reply:
Does anyone have a preference on listing the steps or pointing them to the install instructions?

Reply:
going to leave it as referincing the install steps from docker